protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -264,6 +267,18 @@ definitions:
       status:
         description: "The current status of the stream"
         "$ref": "#/definitions/AirbyteStreamStatus"
+  AirbyteAnalyticsTraceMessage:
```

Comment:
as an airbyte employee looking at this file, where do I go to understand what this message does/how it works? can you add a description to this?